web/client/utils/CoordinatesUtils.js

```diff
@@ -1011,7 +1011,9 @@ export const makeBboxFromOWS = (lcOWS, ucOWS) => {
 
     return [lc[0], lc[1], uc[0], uc[1]];
 };
-
+export const getPolygonFromCircle = (center, radius, units = "meters", steps = 100) => {
```

Comment:
This change is not releated to the PR title. Could you split into another PR or at least to a different commit with a meaningful message? I would also add a test for it.

Reply:
you caught me :)